README.textile

```diff
@@ -150,13 +150,13 @@ curl -XPUT 'http://localhost:9200/kimchy/_doc/2?pretty' -H 'Content-Type: applic
 
 The above will index information into the @kimchy@ index. Each user will get their own special index.
 
-Complete control on the index level is allowed. As an example, in the above case, we would want to change from the default 5 shards with 1 replica per index, to only 1 shard with 1 replica per index (== per twitter user). Here is how this can be done (the configuration can be in yaml as well):
+Complete control on the index level is allowed. As an example, in the above case, we might want to change from the default 1 shard with 1 replica per index, to 2 shards with 1 replica per index (== per twitter user). Here is how this can be done (the configuration can be in yaml as well):
```

Comment:
Maybe instead of `"[...] (== per twitter user)"` we say something like `"[...] (because we expect this user to be larger)"`?

E.g. I think the old example was trying to say that if you have an index per user, you'd want it to be a single shard.  So that doesn't quite apply in the new example, but would be the opposite... you think the user might be larger so it needs more shards.

I might be misinterpreting it though, optional comment :)

Reply:
This is a good point and I've changed this parenthetical along those lines. Thanks!